src/agents/plots_videos.jl

```diff
@@ -191,7 +199,11 @@ function abm_video(file, model, agent_step!, model_step! = Agents.dummystep;
     end
 
     fig = Figure(; resolution, backgroundcolor = DEFAULT_BG)
-    ax = fig[1,1][1,1] = Axis(fig; title = t, titlealign = :left, axiskwargs...)
+    ax = fig[1,1][1,1] = if length(size(model.space.s)) == 3
+        Axis3(fig; title = t, titlealign = :left, axiskwargs...)
+    else
+        Axis2(fig; title = t, titlealign = :left, axiskwargs...)
```

Comment:
Does this explicitly need to be `Axis2` or can it be `Axis`?

Reply:
It should be `Axis`, `Axis2` is an accident